plugins/action/zos_script.py

```diff
@@ -56,7 +56,9 @@ def run(self, tmp=None, task_vars=None):
         if not remote_src:
             script_path = path.abspath(path.normpath(script_path))
             script_name = path.basename(script_path)
-            tmp_path = module_args.get('tmp_path')
+            # Accessing the globally-defined temporary directory
+            # that Ansible expects to be used.
+            tmp_path = self._connection._shell._options.get("remote_tmp")
```

Comment:
i think we briefly talked about this over slack, but just to re-reconfirm, is there a base/default value here? is there any chance that the call will return `None` or something similar which could set that `tmp_path` to something invalid and cause issues later on in the module execution?

Reply:
There's a default value that Ansible sets when the user doesn't specify anything in config files or environment variables: `~/.ansible/tmp`.